docs/design/solutions/hub-and-spoke-tpe/flow_delete_uninstall.txt

```diff
@@ -0,0 +1,48 @@
+@startuml
+title FLOW delete
+
+participant Client
+participant NorthBound as NB
+control kilda.northbound
+participant "FLOW SplitterBolt" as SplitterBolt
+participant "FLOW CrudBolt" as CrudBolt
+participant "FLOW ErrorBolt" as ErrorBolt
+participant "FLOW NorthboundReplyBolt" as NorthboundReplyBolt
+participant "FLOW FlowCommandBolt" as FlowCommandBolt
+participant FlowService
+participant DAO
+participant "FLOW TransactionBolt" as TransactionBolt
+control kilda.flow
+participant "FLOW SpeakerBolt" as SpeakerBolt
+
+Client -> NB: /flows DELETE
+NB -> kilda.flow: CommandMessage\FlowDeleteRequest
+kilda.flow -> SplitterBolt: CommandMessage\FlowDeleteRequest
+SplitterBolt -> CrudBolt: CommandMessage\FlowDeleteRequest
+CrudBolt -> FlowCommandBolt: InfoMessage.FlowInfoData
+CrudBolt -> NorthboundReplyBolt:  FlowInfoData
+alt error
+	CrudBolt -> ErrorBolt: ErrorMessage
+	ErrorBolt -> NorthboundReplyBolt: ErrorMessage
+end
+NorthboundReplyBolt -> kilda.northbound: InfoMessage or\n ErrorMessage
+
+loop in forward, reveres
+  loop in segments
+    FlowCommandBolt -> FlowService: delete flow
+    FlowService -> DAO: delete flow
+    note right of DAO: status is In_progress
```

Comment:
"in progress"? How are you going to distinguish "in progres" in flow create and "in progress" in flow delete?